test/px-map-vector-layer-tests.js

```diff
@@ -0,0 +1,115 @@
+document.addEventListener("WebComponentsReady", function() {
+  runCustomTests();
+});
+
+function runCustomTests() {
+
+  describe('px-map-vector-layer', function () {
+    let vectorLayerFixture;
```

Comment:
Unfortunately we can't use let/const, arrow functions, or other ES6/ES2015 niceties in our test suite because we don't transpile our tests before running them on non-ES6 compatible browsers. Sadly we must stick to IE11+ compatible JS.